test/WorkflowCore.Tests.Sqlite/SqliteCollection.cs

```diff
@@ -10,7 +10,7 @@ public class SqliteCollection : ICollectionFixture<SqliteSetup>
 
     public class SqliteSetup : IDisposable
     {
-        public string ConnectionString { get; set; }
+        public static string ConnectionString { get; set; }
```

Comment:
Why does this need to be made static?

Reply:
I wanted to have simple access to the value from the new `SqliteStopScenario` as well as aligning it with the way all the other scenario setups have static connection strings.